lmdeploy/turbomind/engine_config.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from pydantic.dataclasses import dataclass
+
+
+@dataclass
+class EngineConfig:
+    model_name: str = None
```

Comment:
could we add docstring for these class attributes to let users know the meaning of each attrbute?
some thing like
```
"""Turbomind engine config

Attributes:
    model_name (str): xxx
    ......
"""
```